api/api.go

```diff
@@ -271,6 +272,21 @@ func (api *API) listAlerts(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	if silencedParam := r.FormValue("silenced"); silencedParam != "" {
```

Comment:
You could potentially use `strconv.ParseBool`, but I would ask for @fabxc 's preference here, as I'm not sure what's common in the go community. It's a pretty easy check when it's a string.